services/beaconchain/src/main/java/tech/pegasys/artemis/services/beaconchain/BeaconChainController.java

```diff
@@ -268,6 +275,13 @@ private void initEth1DataCache() {
         });
   }
 
+  private void initSyncStateTracker() {
+    LOG.debug("BeaconChainController.initSyncStateTracker");
+    // TODO: Make startup conditions configurable and set reasonable defaults based on the network
+    syncStateTracker =
+        new SyncStateTracker(asyncRunner, syncService, p2pNetwork, 5, Duration.ofSeconds(10));
```

Comment:
As you mentioned in the description making the last two arguments Constants with clear names would be nice.

Reply:
Yep they're command line options with defaults provided by the `--network` choice now.